### PR TITLE
fix test drop table exceptions

### PR DESCRIPTION
**Pull request by ConstNW**
change `DROP TABLE` to `DROP TABLE IF EXISTS`

### CODE REVIEW

File: test/CommonDatabaseTest.hx [hunk 1]
```diff
@@ -32,7 +32,7 @@ class CommonDatabaseTest extends utest.Test {
 			var quoteField = @:privateAccess cls.manager.quoteField;
 			var name = cls.manager.dbInfos().name;
 			try {
-				Manager.cnx.request('DROP TABLE ${quoteField(name)}');
+				Manager.cnx.request('DROP TABLE IF EXISTS ${quoteField(name)}');
```
Comment (jonasmalacofilho) [inline test/CommonDatabaseTest.hx]: There isn't a major problem in doing this, but *why is it necessary?*

SQL errors due to inexistent tables should already be cleanly swallowed by the `catch` bellow.

Comment (ConstNW) [reply test/CommonDatabaseTest.hx]: It's stop useless exception raises which makes my annoy at debugging code on hl target

Comment (jonasmalacofilho) [reply test/CommonDatabaseTest.hx]: Fair enough, but then the try-catch should be removed as well.

**Comment by jonasmalacofilho** [comment]
Thanks!

**Pull request merged**